integration/simulation/l2_network.go

```diff
@@ -38,30 +55,13 @@ func (cfg *L2NetworkCfg) BroadcastTx(tx nodecommon.EncryptedTx) {
 
 	for _, a := range cfg.nodeAddresses {
 		address := a
+		// we want to control the delay, so we use the send function
 		obscurocommon.Schedule(cfg.delay()/2, func() {
-			broadcastBytes(address, msgEncoded)
+			cfg.p2p.SendBytes(address, msgEncoded)
```

Comment:
why can't we use  "p2p.BroadcastTx" here?

Reply:
Because we want to control the broadcast with the `obscurocommon.Schedule(cfg.delay()/2, func() {`